CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+# 1.0.0
+* Removes unnecessary requirements for loading snapshots. (#433)
```

Comment:
This was a bug fix for a feature added in this release so can probably exclude it.